common/src/main/java/com/wynntils/features/overlays/CustomPlayerListFeature.java

```diff
@@ -46,7 +47,8 @@ public class CustomPlayerListFeature extends Feature {
 
     @SubscribeEvent
     public void onRender(RenderEvent.Pre event) {
-        if (event.getType() == RenderEvent.ElementType.PLAYER_TAB_LIST) {
+        if (event.getType() == RenderEvent.ElementType.PLAYER_TAB_LIST
```

Comment:
That's interesting. Why do disabled features get events? 🤔

Reply:
If you only disable the overlay...

Reply:
Ah, I see... Ok, then this is fine.

Reply:
(But maybe we should do something about the way we end up with "overlay-wrapping features"; somehow promote the overlay themselves to proper features?)